CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## v1.9.0
+
+This release focuses on bug fixes and developer tooling (e.g. vagrant) and UX improvements.
```

Comment:
Vagrant changes are not developer tooling, current Vagrantfile is meant to be used by users during first trying-out.